web/README.md

```diff
@@ -0,0 +1,418 @@
+# HarbourBridge: Web APIs (Experimental)
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
+using data from an existing PostgreSQL or MySQL database. This README provides
+details of the tool's Web APIs capabilities, Web APIs allows user to visualize the conversion
+using UI and interact with the tool to customize some of the conversion. For general HarbourBridge information
+see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-turnkey-spanner-evaluation). Note that this feature is experimental, and should not be used for production database.
+
+### Starting web server for HarbourBridge
+
+The following example assume `harbourbridge` has been added to your PATH
+environment variable.
+
+HarbourBridge's Web API feature can be used with all the driver modes available, using mysql or postgres dump or direct connection.
+
+To start HarbourBridge web server, run:
+
+```sh
+harbourbridge --web
+```
+
+The tool will be available on port 8080
+
+You can go to `<base-url>/` to visit the UI of the tool.
+
+    eg: localhost:8080/
+
+For more details on how to use the UI, you can visit: `<base-url>/userManual.html`.
+
+    eg: localhost:8080/userManual.html
+
+<ins>**Note:**</ins>
+
+Data migration is not supported when using `pg_dump` or `mysqldump` driver options when the converted schema contains Interleaved tables. It is recommended to use `postgres` or `mysql` driver for data migration with schema having Interleaved tables.
```

Comment:
Suggest rephrasing:

The `pg_dump` and `mysqldump` drivers cannot be used for data migration if the
Spanner schema has interleaved tables. Note that the `postgres` or `mysql` drivers
do not have this restriction -- consider using these as an alternative.

Reply:
updated statement